solver/llbsolver/ops/exec.go

```diff
@@ -141,6 +145,14 @@ func (e *execOp) CacheMap(ctx context.Context, g session.Group, index int) (*sol
 			ComputeDigestFunc solver.ResultBasedCacheFunc
 			PreprocessFunc    solver.PreprocessFunc
 		}, e.numInputs),
+		Opts: solver.CacheOpts(map[interface{}]interface{}{
```

Comment:
Why is this is ops level if it doesn't seem to be using anything from the ops implementation? Lmk if you wish to fix in follow-up.

Reply:
Update in follow-up if needed

Reply:
I see your point and we could move this up to be in the `CacheMap` method in `solver/jobs.go` but it's a bit weird in that `llbsolver/ops/merge.go` and `llbsolver/ops/diff.go` actually need access to the progress group in order to pass them to cache manager during exec, so it would requiring some refactoring or accepting that it gets set in different parts of the code depending on the op type... I'll look at it quickly and see if there's any clean paths though.